apps/chef_objects/src/chef_client.erl

```diff
@@ -127,26 +116,34 @@ set_updated(#chef_client{} = Object, ActorId) ->
     Now = chef_object_base:sql_date(now),
     Object#chef_client{updated_at = Now, last_updated_by = ActorId}.
 
+create_query(#chef_client{server_api_version = ?API_v0}) ->
```

Comment:
Isn't this going to default to using the new stuff if the client doesn't send up the API version it wants?